spec/units/commands/git_command_spec.rb

```diff
@@ -53,5 +59,55 @@
         env,
       )
     end
+
+    context 'with autocorrect enabled' do
```

Comment:
Yes, this was handled in the old parser. I couldn't work out how to handle this in the new parser so this seemed like the next best thing.